docs/our-approach-to-data.md

```diff
@@ -8,15 +8,15 @@ There have been three major "eras" of data management throughout the lifetime of
 
 ### First Era: Emitter Objects (June 2014 - April 2015)
 
-Our original approach to managing data took an object-oriented approach, wherein an instance of the store would inherit the [`EventEmitter` interface](https://nodejs.org/api/events.html#events_class_eventemitter). Typically, a single instance of each object store was shared across the entire application. The instance was responsible for storing data, but included conveniences to automatically fetch data upon the first request. Used in combination with the [`data-observe` mixin](../client/lib/mixins/data-observe), a developer could monitor an instance of the store passed as a prop to a React component to automatically re-render its contents if the store emitted a `change` event.
+Our original approach to managing data took an object-oriented approach, wherein an instance of the store would inherit the [`EventEmitter` interface](https://nodejs.org/api/events.html#events_class_eventemitter). Typically, a single instance of each object store was shared across the entire application. The instance was responsible for storing data, but included conveniences to automatically fetch data upon the first request. Used in combination with a `data-observe` mixin, a developer could monitor an instance of the store passed as a prop to a React component to automatically re-render its contents if the store emitted a `change` event.
 
 **Identifying characteristics:**
 
 - Module directories in `lib`, suffixed with `-list`
 - Index file exports a common shared instance of the object prototype
 - A `list.js` file includes the store prototype
 - The list contains a `get` method which triggers a fetch if no data exists
-- Used with the [`data-observe` mixin](../client/lib/mixins/data-observe) in a React component
+- Used with a `data-observe` mixin in a React component
```

Comment:
What do you think: should we add a link to this PR here for context? 😉  It will be valuable 2 years from now when nobody has heard about the `data-observe` mixin (and I hope, mixins as a whole 😉 )

Reply:
I think it's easy to find info like this in the repo history. In a document like this, I think it would be buried and nobody would appreciate it 🙂

Reply:
👍  I don't mind your decision at all. 😉